spec/controllers/subscription_spec.rb

```diff
@@ -20,7 +20,10 @@
     let(:subscription) do
       Routemaster::Models::Subscription.new(
         subscriber: 'charlie'
-      )
+      ).tap do |sub|
```

Comment:
Why is it necessary to tap here as opposed to using something like:

```ruby
Routemaster::Models::Subscription.new(
  subscriber: 'charlie',
  callback: 'https://example.com/events',
  uuid: 's3cr3t'
)
```

I've seen this pattern to get around mass assignment, but I'm sure I don't know all the reasons for using it.